spec/fixtures/files/azure_ad_rsa_private.pem

```diff
@@ -0,0 +1,27 @@
+-----BEGIN RSA PRIVATE KEY-----
```

Comment:
This is a used in mocking the azure_ad api, and was generated specifically for that purpose. I'll take a look at doing it dynamically to avoid any questions!